sdk/keyvault/azure-security-keyvault-secrets/test/sample/test_app.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#if defined(_MSC_VER)
+#define _CRT_SECURE_NO_WARNINGS
+#endif
+
+#if defined(BUILD_CURL_HTTP_TRANSPORT_ADAPTER)
+#include <curl/curl.h>
+#endif
+
+#include <azure/identity.hpp>
+#include <azure/keyvault/secrets/secret_client.hpp>
+
+using namespace Azure::Security::KeyVault::Secrets;
+
+int main()
+{
+  auto tenantId = std::getenv("AZURE_TENANT_ID");
+  auto clientId = std::getenv("AZURE_CLIENT_ID");
+  auto clientSecret = std::getenv("AZURE_CLIENT_SECRET");
+  auto credential
+      = std::make_shared<Azure::Identity::ClientSecretCredential>(tenantId, clientId, clientSecret);
+
+  SecretClient secretClient(std::getenv("AZURE_KEYVAULT_URL"), credential);
+
+  auto response = secretClient.GetSecret("testSecret");
```

Comment:
You might want to check the status code and return > 0 if the request failed or something.
Right now, if the server returns an error, we don't know.

Also, if you are using this sample for making your own testing, you can also set the ENV VAR `AZURE_LOG_LEVEL=1` (you might know already about it).  It will print the request and response to the console

Reply:
ATM i'm using it purely for debugging, i'll make sample apps later on